tests/cypress/dash/v_data_loading.py

```diff
@@ -59,24 +61,27 @@
 
 @app.callback(
     Output("table", "style_cell_conditional"),
-    [Input("change-property", "value")]
+    [Input("change-other-property", "value")]
 )
 def dontTriggerWait(to_change):
-    if to_change == 'dont_change_data':
-        sleep(5)
+    if to_change != 'dont_change_data':
+        raise PreventUpdate
+
+    sleep(5)
     return []
 
 
 @app.callback(
     Output("table", "data"),
-    [Input("change-property", "value")],
-    [State("table", "data")]
+    [Input("change-data-property", "value")]
 )
 # pylint: disable=unused-argument
-def triggerWait(to_change, current):
-    if to_change == 'change_data':
-        sleep(5)
-    return current
+def triggerWait(to_change):
+    if to_change != 'change_data':
+        raise PreventUpdate
+
+    sleep(5)
+    return df[0:250]
```

Comment:
Remove the need for `data` as state by returning the baseline. copy/paste already tests that state/output data works